server/utils/excelExport/poamExport.ts

```diff
@@ -374,7 +378,31 @@ export async function generatePoam(
   const recommendations = 20;
 
   const results = await getEvaluationSummary(boundaryId, undefined, true);
```

Comment:
The 'undefined' in the middle looks kinda wonky.  I would recommend leaving the required parameter as it is (boundaryId), but moving the other two parameters to an `options` (or whatever you wanna call it) object that would have those parameters as optional attributes.

This would change this function call to look like `getEvaluationSummary(boundaryId, {poam: true})`.